static/app/gettingStartedDocs/powershell/powershell.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import {renderWithOnboardingLayout} from 'sentry-test/onboarding/renderWithOnboardingLayout';
+import {screen} from 'sentry-test/reactTestingLibrary';
+import {textWithMarkupMatcher} from 'sentry-test/utils';
+
+import docs from './powershell';
+
+describe('powershell onboarding docs', function () {
+  it('renders docs correctly', async function () {
+    renderWithOnboardingLayout(docs, {
+      releaseRegistry: {
+        'sentry.powershell': {
```

Comment:
see: https://github.com/getsentry/sentry-release-registry/pull/150

```suggestion
        'sentry.dotnet.powershell': {
```